### PR TITLE
#1305 Document version 1.29

### PR DESCRIPTION
**Pull request by kylekatarnls**

### CODE REVIEW

File: docs/index.src.html [hunk 1]
```diff
@@ -2138,6 +2138,26 @@ <h1 id="api-period">CarbonPeriod</h1>
...
+
+<p>Just as <a href="#api-macro">Carbon</a> and <a href="#api-interval">CarbonInterval</a>, CarbonPeriod has support for macros.</p>
+
+<p><pre><code class="php">{{::lint(
+CarbonPeriod::macro('countWeekdays', function ($self = null) {
+    return $self->filter('isWeekday')->count();
+});
+)}}
+{{cpmacro1::exec(echo CarbonPeriod::create('2017-11-01', '2017-11-30')->countWeekdays();/*pad(72)*/)}} // {{cpmacro1_eval}}
+{{cpmacro2::exec(echo CarbonPeriod::create('2017-12-01', '2017-12-31')->countWeekdays();/*pad(72)*/)}} // {{cpmacro2_eval}}
```
Comment (kylekatarnls) [inline docs/index.src.html]: Duplicates macro chapter

Comment (imbrish) [reply docs/index.src.html]: Link to that chapter instead?

**Comment by imbrish** [comment]
Ready to go 👍

File: docs/index.src.html [hunk 2]
```diff
@@ -2074,10 +2074,10 @@ <h1 id="api-period">CarbonPeriod</h1>
 <p>Order in which filters are added can have an impact on the performance and on the result, so you can use <code>addFilter()</code> to add a filter in the end of stack; and you can use <code>prependFilter()</code> to add one at the beginning. You can even use <code>setFilters()</code> to replace all filters. Note that you'll have to keep correct format of the stack and remember about internal filters for recurrences limit and end date. Alternatively you can use <code>resetFilters()</code> method and then add new filters one by one.</p>
 
 <p>For example, when you add a custom filter that limits the number of attempted dates, the result will be different if you add it before or after the weekday filter.</p>
+
 <p><pre><code class="php">{{::lint(
-$period = CarbonPeriod::between('2018-05-03', '2018-05-25')->filter(function ($date) {
-    return $date->isWeekday();
-});
+// Note that you can pass as string any Carbon method including macros starting with "is"
```
Comment (imbrish) [inline docs/index.src.html]: This comment is a bit confusing, what about "Note that you can pass a name of any Carbon method starting with "is", including macros"?

**Pull request merged**